content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -1,22 +1,156 @@
 +++
-title = "Kubeflow Deployment with kfctl_existing_arrikto"
-description = "Instructions for installing Kubeflow with kfctl_existing_arrikto config"
+title = "Multi-user, auth-enabled Kubeflow with kfctl_existing_arrikto"
+description = "Instructions for installing Kubeflow with kfctl_existing_arrikto.yaml config"
 weight = 4
 +++
 
-This deployment uses [Dex](https://github.com/dexidp/dex) and [Istio](https://istio.io/) for vendor-neutral authentication.
+Follow these instructions if you want to install Kubeflow on an existing Kubernetes cluster.
 
-**Maintainer and supporter: Arrikto**
+This installation of Kubeflow is maintained by
+[Arrikto](https://www.arrikto.com/), it is geared towards existing Kubernetes
+clusters and does not depend on any cloud-specific feature.
+
+In this reference architecture, we use [Dex](https://github.com/dexidp/dex) and
+[Istio](https://istio.io/) for vendor-neutral authentication.
+
+This deployment works well for on-prem installations, where companies/organizations need LDAP/AD integration for multi-user authentication, and they don't want to depend on any cloud-specific feature.
+
+![kfctl_existing_arrikto_architecture](../../kfctl_existing_arrikto-architecture.svg)
+
+Read the relevant [article](https://journal.arrikto.com/kubeflow-authentication-with-istio-dex-5eafdfac4782) for more info about this architecture.
 
-![platform existing architecture](https://i.imgur.com/OlaN73j.png)
 
 ### Prerequisites
-- Kubernetes Cluster with LoadBalancer support. Refer [Load Balancer](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/) guide.
 
+- Kubernetes Cluster with LoadBalancer support.
+
+If you don't have support for LoadBalancer on your cluster, please
+follow the instructions below to deploy MetalLB in [Layer 2
+mode](https://metallb.universe.tf/configuration/#layer-2-configuration).
```

Comment:
Recommendation:

```
follow the instructions below to deploy MetalLB in Layer 2
mode. (You can read more about Layer 2 mode in the
[MetalLB docs](https://metallb.universe.tf/configuration/#layer-2-configuration).)
```

The reason for this suggestion is that my first inclination was to click the link and look for installation instructions there. The word "below" is ambiguous, unless we make it clear that the link leads to extra info.

Reply:
Thank Sarah, this makes sense